search_service/api/document.py

```diff
@@ -78,11 +79,15 @@ def put(self) -> Tuple[Any, int]:
         :param data: list of data objects to be indexed in Elasticsearch
         :return: name of index
         """
-        self.parser.add_argument('data', required=True)
+        self.parser.add_argument('data', required=True, action='append')
         args = self.parser.parse_args()
 
         try:
-            data = self.schema(many=True, strict=False).loads(args.get('data')).data
+            table_dict_list = []
+            for table_str in args.get('data'):
+                table_dict_list.append(literal_eval(table_str))
```

Comment:
you could also simplify the above as `table_objs = [literal_eval(table_str) for table_str in args.get('data')]`.

Reply:
does the endpoint work for user,table,dashboard as well?  would be good to add a todo for things that are missing.

Reply:
For user, we already have a different endpoint, for dashboard, we need to think about it, not sure whether the fields are the same as table, if not, we can also create a new one.